### PR TITLE
Include new function into Bridge's ABI.

### PR DESCRIPTION
**Pull request by pamgonzalez**
Include the function registerBtcTransfer into the ABI from the Bridge contract.

### CODE REVIEW

File: abis/bridge.json [hunk 1]
```diff
@@ -780,6 +780,51 @@
...
+      {
+        "name": "lbcAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lpbtcAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lpbtcAddress",
```
Comment (josedahlquist) [inline abis/bridge.json:817]: This name is not correct

File: abis/bridge.json [hunk 2]
```diff
@@ -780,6 +780,51 @@
...
+      {
+        "name": "userRefundAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lbcAddress",
+        "type": "string"
+      }, 
+      {
+        "name": "lpbtcAddress",
```
Comment (josedahlquist) [inline abis/bridge.json:813]: Please use more significant names

File: abis/bridge.json (hunk 1 above)
Comment (guidohernan93) [reply abis/bridge.json:817]: Should be `shouldTransferToContract` I think

File: abis/bridge.json [hunk 3]
```diff
@@ -780,6 +780,51 @@
...
+      {
+        "name": "derivationArgumentsHash",
+        "type": "bytes32"
+      },
+      {
+        "name": "userRefundBtcAddress",
+        "type": "bytes"
+      }, 
+      {
+        "name": "LiquidityBridgeContractAddress",
```
Comment (guidohernan93) [inline abis/bridge.json:809]: should be `liquidityBridgeContractAddress`

File: abis/bridge.json [hunk 4]
```diff
@@ -780,6 +780,51 @@
...
+        "type": "string"
+      }, 
+      {
+        "name": "LiquidityProviderBtcAddress",
```
Comment (guidohernan93) [inline abis/bridge.json:813]: should be `liquidityBridgeContractAddress`

**Review by guidohernan93 — APPROVED** [review]

**Review by josedahlquist — APPROVED** [review]

**Pull request merged**